packages/components/src/__future__/Tag/_docs/Tag.stories.tsx

```diff
@@ -1,94 +1,52 @@
 import React from "react"
-import { Meta, StoryFn } from "@storybook/react"
-import { TagIcon } from "~components/Icons"
-import { ComponentDocsTemplate } from "~storybook/components/DocsContainer"
-import { StickerSheet } from "~storybook/components/StickerSheet"
-import { Tag } from ".."
+import { Meta, StoryObj } from "@storybook/react"
+import { AcademyIcon, ActionOffIcon, AddIcon, TagIcon } from "~components/Icons"
+import { Tag, TagColorKeys } from ".."
+import styles from "./TagStories.module.scss"
 
 export default {
-  tags: ["autodocs"],
-  title: "KAIO-staging/Tag",
+  title: "Components/Tag",
   component: Tag,
-  parameters: {
-    docs: {
-      container: ComponentDocsTemplate,
-    },
-    isInKaio: true,
-    installation: [
-      "yarn add @kaizen/components",
-      "import { Tag } from `@kaizen/components/future`",
-    ],
-    resourceLinks: {
-      sourceCode:
-        "https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__future__/Tag",
-      figma:
-        "https://www.figma.com/file/hANEprZKom5Br1IBXhE8mr/%F0%9F%A7%AA-Kaizen-Labs?type=design&node-id=28588-183332&mode=design&t=VHZDCTJUpaon8PXg-0",
-    },
-  },
 } satisfies Meta<typeof Tag>
 
-export const Playground: StoryFn<typeof Tag> = args => (
-  <Tag {...args}>My tag</Tag>
-)
+export const Playground: StoryObj = {
+  render: args => <Tag {...args}>My tag</Tag>,
+}
+
+export const SimpleTag: StoryObj = {
+  render: args => <Tag {...args}>My Tag</Tag>,
+}
 
-type ListProps = {
-  children: React.ReactNode
+export const Children: StoryObj = {
+  render: args => <Tag {...args}>This text is the children</Tag>,
 }
```

Comment:
This is the format for overriding any default args (extends the ones defined in meta).
`render` has a default definition of (where `Component` is what you have defined in `meta.component`) `args => <Component {...args />`, thus you do not need to define it unless it does not suit your needs.

```suggestion
export const Children: StoryObj = {
  args: {
    children: "This text is the children",
  }
}
```